stats/stats.go

```diff
@@ -56,6 +57,7 @@ func StartStatsServerWithPort(startupOpts StartupOptions, addhandlers ...func(mu
 	}
 
 	logconfig := zap.NewProductionConfig()
+	logconfig.EncoderConfig.EncodeTime = zapcore.ISO8601TimeEncoder
```

Comment:
Do we want to instead just pass in a logger to use? That'll ensure it's always consistent with whoever uses this (someone may not be using the ISO8601TimeEncoder)